pkg/util/mount/mount_linux.go

```diff
@@ -492,6 +492,19 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 		}
 	}
 
+	// Run e2fsck and resize2fs to extends ext2/ext3/ext4 disk size
```

Comment:
if this only works for cinder, you need add some `if` statement or in volume/cinder module.

Reply:
formatAndMount function use by fc,iscsi,photon...